.github/workflows/release-files.yml

```diff
@@ -30,81 +30,73 @@ jobs:
           name: ${{ github.ref_name }}
           draft: false
           prerelease: false
-          generate_release_notes: false
+          generate_release_notes: true
 
   build_release:
     name: Build Release
     needs: create_release
     environment: build
+    outputs:
+      output_linux_amd: ${{ steps.release_package.outputs.output_LINUX_AMD_SHA }}
+      output_linux_arm: ${{ steps.release_package.outputs.output_LINUX_ARM_SHA }}
+      output_mac_amd: ${{ steps.release_package.outputs.output_MAC_AMD_SHA }}
+      output_mac_arm: ${{ steps.release_package.outputs.output_MAC_ARM_SHA }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, macos-15]
         include:
           - os: ubuntu-latest
-            release_suffix: ubuntu
+            release_suffix: ubuntu-amd
```

Comment:
It might be better to change the release suffix to `linux-...` instead of `ubuntu-...`, unless it's really Ubuntu specific.